code/game/objects/structures/fluff.dm

```diff
@@ -261,3 +267,12 @@
 
 /obj/structure/fluff/hedge/opaque //useful for mazes and such
 	opacity = 1
+
+/obj/structure/fluff/tombofhorrors
+	name = "great green devil"
+	desc = "On the wall before you is a relief statue of a devil formed of mosaic plastics. The face has a huge O of a mouth, inside of which the space is dead black darkness."
```

Comment:
add some way to interact with this, even if it's just getting your hand annihilated because you stuck it in the devil's mouth like a dumbass

Reply:
Considered.